### PR TITLE
Fix tests badge

### PR DESCRIPTION
**Pull request by alanhamlett**

### CODE REVIEW

**Comment by gandarez** [comment]
We had a discussion about which branch should actually be on top [here](https://github.com/wakatime/wakatime-cli/pull/338).

**Comment by alanhamlett** [comment]
@gandarez using develop branch now, so we can show status of alpha and stable builds.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # WakaTime CLI
 
-![Tests](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Create%20Release/release?label=%20tests) ![Build](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Build%20and%20upload%20release%20assets) [![Coverage Status](https://coveralls.io/repos/github/wakatime/wakatime-cli/badge.svg?branch=release)](https://coveralls.io/github/wakatime/wakatime-cli?branch=release)
+![Tests](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Tests/develop?label=tests)
+![Build](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Build%20and%20upload%20release%20assets)
```
Comment (gandarez) [inline README.md]: Since we unified workflows it has changed. You need to rebase and get the new one.

**Review by gandarez — COMMENTED** [review]
Are we going to show tests on develop branch and coverage for release branch? It might confuse a bit IMO.

File: README.md (hunk 1 above)
Comment (alanhamlett) [reply README.md]: Here's the list of current workflows we can choose from:

https://api.github.com/repos/wakatime/wakatime-cli/actions/workflows

Comment (alanhamlett) [reply README.md]: We only have one workflow now `Tests` so we should remove the Build badge.

**Review by dron22 — APPROVED** [review]
LGTM :+1:

**Pull request merged**